### PR TITLE
added cross-building

### PR DESCRIPTION
**Pull request by green-creeper**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -28,7 +28,7 @@ lazy val commonConfigs = Seq.empty
 
 lazy val commonSettings = Seq(
   organization := "com.advancedtelematic",
-  scalaVersion := "2.11.8",
+  crossScalaVersions := Seq("2.11.8", "2.12"),
```
Comment (koshelev) [inline build.sbt]: Why not 2.11.11 and 2.12.2?

File: build.sbt [hunk 2]
```diff
@@ -28,7 +28,7 @@ lazy val commonConfigs = Seq.empty
 
 lazy val commonSettings = Seq(
   organization := "com.advancedtelematic",
-  scalaVersion := "2.11.8",
```
Comment (koshelev) [inline build.sbt]: Are you sure it is not needed?

File: build.sbt (hunk 1 above)
Comment (green-creeper) [inline build.sbt]: I made 2.11.8 because it was scalaVersion := "2.11.8"

File: build.sbt [hunk 3]
```diff
@@ -29,6 +29,7 @@ lazy val commonConfigs = Seq.empty
 lazy val commonSettings = Seq(
   organization := "com.advancedtelematic",
   scalaVersion := "2.11.8",
```
Comment (koshelev) [inline build.sbt]: let's make it 2.11.11 :)

Comment (green-creeper) [inline build.sbt]: @koshelev done

**Pull request merged**